lib/chef/resource/apt_package.rb

```diff
@@ -52,6 +52,21 @@ class AptPackage < Chef::Resource::Package
         options '--no-install-recommends'
       end
       ```
+
+      **Prevent the apt_package resource from installing packages with pattern matching names**:
+
+      By default, the apt_package resource will install the named package.
+      If it can't find a package with the exact same name, it will treat the package name as regex string and match with any package that matches that regex.
```

Comment:
We can't use the term `regex` in docs.

```suggestion
      If it can't find a package with the exact same name, it will treat the package name as regular expression string and match with any package that matches that regular expression.
```